src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -323,7 +338,6 @@ def _configure_db_create_params(
     if create_mode != CreateMode.default or engine != Engine.db:
         arg_ctx.ignore('sample_name')
         arg_ctx.ignore('catalog_collation')
-        arg_ctx.ignore('read_scale')
```

Comment:
You should still ignore read_scale and read_replica count if engine == dw (section below)

Reply:
Done

Reply:
Fixed.